cloud_experts_tutorials/cloud-experts-dynamic-certificate-custom-domain.adoc

```diff
@@ -271,7 +283,8 @@ NAME                     READY   AGE
 letsencrypt-production   True    47s
 ----
 
-== Creating a custom domain ingress controller
+[id="cloud-experts-dynamic-certificate-custom-domain-create-cd-ingress-con"]
+== Creating a custom domain Ingress Controller
 
 . Create a new project:
 +
```

Comment:
Either line 283 is missing the commands to run for this step, or the code listing starting on line 291 is both creation of the certificate resource and configuration of the certificate using that resource - pretty sure it's the latter, so line 283 and 289 should be combined

Reply:
@laubai I'm a bit confused by this comment- line 283 is part of the output to confirm the `ClusterIssuer` resource is ready. Line 289 is a command to create a new custom domain Ingress Controller.

Reply:
Sorry, I was looking at the "before" for line numbers for some reason.

Line 296 doesn't appear to have a step in it, only a note:
```
. Create a certificate resource to provision a certificate for the custom domain Ingress Controller:
+
[NOTE]
====
The following example uses a single domain certificate. SAN and wildcard certificates are also supported.
====
```

The next step at line 302 appears to be both creating and configuring the certificate resource.
```
. Configure the certificate:
+
[source,terminal]
----
$ cat << EOF | oc apply -f -
apiVersion: cert-manager.io/v1
kind: Certificate
metadata:
  name: custom-domain-ingress-cert
  namespace: custom-domain-ingress
spec:
  secretName: custom-domain-ingress-cert-tls
  issuerRef:
     name: letsencrypt-production
     kind: ClusterIssuer
  commonName: "${DOMAIN}"
  dnsNames:
  - "${DOMAIN}"
EOF
----
```

Basically I'm wondering if line 296 should be "Create and configure a certificate resource to ....", and line 302 should be omitted.

Reply:
Yes-that makes a lot of sense-thanks!